gusb/gusb-device.c

```diff
@@ -121,6 +121,8 @@ g_usb_device_set_property (GObject      *object,
 	switch (prop_id) {
 	case PROP_LIBUSB_DEVICE:
 		priv->device = g_value_get_pointer (value);
+		if (priv->device)
```

Comment:
!= NULL please

Reply:
It's also a bit inelegant. Maybe an internal function that unrefs the old priv-device if set, and assigns the private ref?

Reply:
> It's also a bit inelegant. Maybe an internal function that unrefs the old priv-device if set, and assigns the private ref?

Yeah, well being construct-only I didn't feel the need for it, but I did thought of having a sort of `g_set_object` like thingy.